packages/app-admin-ui/client/pages/Signout.js

```diff
@@ -86,24 +70,28 @@ const SignedOutPage = () => {
 
   return (
     <Container>
-      <Alerts />
-      <Box>
-        <KeystoneLogo />
-        <Divider />
-        <Content>
-          {loading ? (
-            <LoadingIndicator />
-          ) : (
-            <Fragment>
-              <p>You have been signed out.</p>
-              <p>
-                <a href={signinPath}>Sign In</a>
-              </p>
-            </Fragment>
-          )}
-        </Content>
-      </Box>
-      <Spacer />
+      {loading ? (
+        <Fragment>
+          <LoadingIndicator css={{ height: '3em' }} size={12} />
+          <Caption>Signing out.</Caption>
+        </Fragment>
+      ) : (
+        <Fragment>
+          <Animation name="pulse" duration="500ms">
+            <CheckIcon css={{ color: colors.primary, height: '3em', width: '3em' }} />
+          </Animation>
+          <Caption>You have been signed out.</Caption>
+          <FlexBox>
+            {/* TODO: is it ok to use '/' as a path here? */}
```

Comment:
Not necessarily, they might not even have a front-end application.

Reply:
I'd remove this as we can't make and assumption and don't want to cross concerns between apps

Reply:
What about a UI hook (and if so, should a Home button be included or excluded by default)?